tools/db_bench_tool.cc

```diff
@@ -1205,7 +1205,7 @@ class Stats {
   uint64_t bytes_;
   uint64_t last_op_finish_;
   uint64_t last_report_finish_;
-  std::unordered_map<OperationType, HistogramImpl,
+  std::unordered_map<OperationType, std::shared_ptr<HistogramImpl>,
```

Comment:
shared_ptr here has extra synchronization costs, so it's not a fair comparison. Why would you make it shared_ptr here at all?